docs/release-announcement.md

```diff
@@ -1,6 +1,21 @@
 # Release announcement
 
+## Azure Industrial IoT Platform Release 2.8.2
+
+We are pleased to announce the release of version 2.8.2 of our Industrial IoT Platform components as a second patch update of the 2.8 Long-Term Support (LTS) release. This release contains important backward compatibility fixes with version 2.5.x, performance optimizations as well as security updates and bugfixes.
+
+> IMPORTANT
+> We suggest to update from  version 2.5 or later to ensure secure operations of your deployment. OPC Publisher 2.8.2 addresses backwards compatibilities issues with version 2.5.x.
+
+### Fundamentals related fixes
+
+- [OPC Publisher] Implemented the backwards compatible [Direct Methods API](docs/modules/publisher-directmethods.md) of 2.5.x publisher. The migration path is documented [here](docs/modules/publisher-migrationpath.md)
```

Comment:
The URLs should not contain `docs/` prefix as this md file itself is already in `docs` folder.

Reply:
Fixed in #1644